client/api/disk/v1beta3/api.proto

```diff
@@ -6,31 +6,31 @@ option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta3
 
 service Disk {
     // ListDiskLocations returns locations <Adapter, Bus, Target, LUN ID> of all
-    // disk devices enumerated by the host
+    // disk devices enumerated by the host.
     rpc ListDiskLocations(ListDiskLocationsRequest) returns (ListDiskLocationsResponse) {}
 
-    // PartitionDisk initializes and partitions a disk device (if the disk has not
-    // been partitioned already) and returns the resulting volume device ID
+    // PartitionDisk initializes and partitions a disk device with the GPT partition style
+    // (if the disk has not been partitioned already) and returns the resulting volume device ID.
     rpc PartitionDisk(PartitionDiskRequest) returns (PartitionDiskResponse) {}
 
-    // Rescan refreshes the host's storage cache
+    // Rescan refreshes the host's storage cache.
     rpc Rescan(RescanRequest) returns (RescanResponse) {}
 
-    // ListDiskIDs returns a map of DiskID objects where the key is the disk number
+    // ListDiskIDs returns a map of DiskID objects where the key is the disk number.
     rpc ListDiskIDs(ListDiskIDsRequest) returns (ListDiskIDsResponse) {}
 
-    // DiskStats returns the stats for the disk
-    rpc DiskStats(DiskStatsRequest) returns (DiskStatsResponse) {}
+    // GetDiskStats returns the stats of a disk.
+    rpc GetDiskStats(GetDiskStatsRequest) returns (GetDiskStatsResponse) {}
```

Comment:
mention currently stat is only include disk size in the comment?

Reply:
done